Publish latest operator build to quay workflow is failing.

**Describe the bug**
It appears that the workflow that publishes the latest operator build to quay has been failing. From looking at the latest push to `master`, the logs suggest that it's because it isn't using the appropriate credentials to talk to quay.

```
Run make docker-push
docker push ***/argocd-operator:latest
The push refers to repository [***/argocd-operator]
29b337577a2a: Preparing
9cfd2c8d9aea: Preparing
636092609a0e: Preparing
6d75f23be3dd: Preparing
unauthorized: access to the requested resource is not authorized
make: *** [Makefile:105: docker-push] Error 1
Error: Process completed with exit code 2.
```

Earlier in the workflow it should authenticate against the docker repository with the following secrets:

```
      - name: Login to Registry
        uses: docker/login-action@v1
        with:
          registry: ${{ secrets.REGISTRY_URL }}
          username: ${{ secrets.REGISTRY_USERNAME }}
          password: ${{ secrets.REGISTRY_PASSWORD }}
```

This is likely because of one of the following:
- These are not the appropriate secret names that we should be using
- They are the appropriate secret names, but don't have the expected values.

**To Reproduce**
Verify that the last push to `master` had a failed workflow
On any subsequent pushes to `master`, this workflow will continue to fail


**Expected behavior**
Each time a push to master occurs, the latest version of the operator should be pushed to the `latest` tag on quay.

**Screenshots**

https://github.com/argoproj-labs/argocd-operator/runs/4216376173?check_suite_focus=true

**Additional context**
N/A

--- a/.github/workflows/publish.yaml
+++ b/.github/workflows/publish.yaml
@@ -10,7 +10,9 @@ jobs:
     runs-on: ubuntu-latest
 
     env:
-      IMG: ${{ secrets.REGISTRY_URL }}/argocd-operator:latest
+      REGISTRY_URL: quay.io
+      ORG: argoprojlabs
+      REPO: argocd-operator
 
     steps:
 
@@ -30,13 +32,17 @@ jobs:
 
       - name: Build operator and Tag Latest
         run: make docker-build
+        env:
+          IMG: ${{ env.REGISTRY_URL }}/${{ env.ORG }}/${{ env.REPO }}:latest
 
       - name: Login to Registry
         uses: docker/login-action@v1
         with:
-          registry: ${{ secrets.REGISTRY_URL }}
+          registry: ${{ env.REGISTRY_URL }}
           username: ${{ secrets.REGISTRY_USERNAME }}
           password: ${{ secrets.REGISTRY_PASSWORD }}
 
       - name: Push latest operator
         run: make docker-push
+        env:
+          IMG: ${{ env.REGISTRY_URL }}/${{ env.ORG }}/${{ env.REPO }}:latest

